### PR TITLE
profitwell Integration

### PR DESCRIPTION
**Pull request by akashrpo**
## Description of the change

> Description here

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> Fix [#1]() 

### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review 

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

### CODE REVIEW

**Review by arajguha — APPROVED** [review]

File: destinations/analytics/profitwell.md [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+A sample `identify` call is as shown:
+
+```javascript
+  rudderanalytics.identify(
+    "user0001",
+    {
+      planId: "Starter",
+      email: "axel@testmail.com",
+      planInterval: "month",
+      effectiveDate: 1630645519,
```
Comment (ameypv-rudder) [inline destinations/analytics/profitwell.md:69]: is there any specific convention/format in which `effectiveDate` must be supplied?

File: destinations/analytics/profitwell.md [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+    }
+  );
+```
+
+{% hint style="info" %}
+Note that:
+
+* The `externalId` of type `profitwellUserId` is mapped to ProfitWell's `user_id` field. 
+* The `externalId` of type `profitwellSubscriptionId` is mapped to ProfitWell's `subscription_id` field. 
+{% endhint %}
```
Comment (ameypv-rudder) [inline destinations/analytics/profitwell.md:91]: Please confirm if the above two points are accurately presented.

File: destinations/analytics/profitwell.md [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+| **RudderStack Field** | **ProfitWell Field**   |
+| :-------------------- | :--------------- |
+| `planId`              | `plan_id`        |
+| `planInterval`        | `plan_interval`  |
+| `value`               | `value`          |
+| `status`              | `status`         |
+| `effectiveDate`       | `effective_date` |
+
+{% hint style="info" %}
+While creating a subscription, if `effectiveDate` is not provided in the `identify` call, then RudderStack takes the date from the event call's `originalTimestamp`.
```
Comment (ameypv-rudder) [inline destinations/analytics/profitwell.md:126]: I rewrote this sentence for more clarity. Please verify.

File: destinations/analytics/profitwell.md [hunk 4]
```diff
@@ -0,0 +1,152 @@
...
+
+{% hint style="info" %}
+For more information on using these fields, refer to the [**ProfitWell documentation**](https://profitwellapiv2.docs.apiary.io/#).
+{% endhint %}
+
+RudderStack discards the `identify` event in the following two scenarios:
+
+* For a given `profitwellUserId`, a user account is not found in ProfitWell.
+* For a given `profitWellUserId`, a `profitwellSubscriptionId` is not found.
+
```
Comment (ameypv-rudder) [inline destinations/analytics/profitwell.md:137]: Please confirm the above two conditions for discarding the event. I rewrote the whole section for better clarity.

File: destinations/analytics/profitwell.md (hunk 1 above)
Comment (akashrpo) [reply destinations/analytics/profitwell.md:69]: It must be provided in UNIX timestamp format.

File: destinations/analytics/profitwell.md (hunk 2 above)
Comment (akashrpo) [reply destinations/analytics/profitwell.md:95]: LGTM

File: destinations/analytics/profitwell.md (hunk 3 above)
Comment (akashrpo) [reply destinations/analytics/profitwell.md:126]: LGTM

File: destinations/analytics/profitwell.md (hunk 4 above)
Comment (akashrpo) [reply destinations/analytics/profitwell.md:137]: LGTM

**Pull request merged**